batched/dense/src/KokkosBatched_Vector_SIMD.hpp

```diff
@@ -22,7 +22,7 @@
 #include <KokkosBatched_Vector.hpp>
 #include "KokkosKernels_Macros.hpp"
 
-#if defined(__CUDA_ARCH__) || defined(__HIP_DEVICE_COMPILE__)
+#if defined(__CUDA_ARCH__) || defined(__HIP_DEVICE_COMPILE__) || defined(SYCL_LANGUAGE_VERSION)
```

Comment:
The corresponding macro would rather be
```suggestion
#if defined(__CUDA_ARCH__) || defined(__HIP_DEVICE_COMPILE__) || defined(__SYCL_DEVICE_ONLY__)
```

Reply:
Done